src/screepsServer.ts

```diff
@@ -12,10 +12,11 @@ const MOD_FILE = 'mods.json';
 const DB_FILE = 'db.json';
 
 interface ScreepServerOptions {
-    path?: string,
-    logDir?: string,
-    port? : number,
-    modFile?: string,
+    path: string,
+    logdir: string,
+    port : number,
+    modfile?: string,
+    mainLoopResetInterval?: number,
```

Comment:
This options is only ever specified in the basic.test.ts. Should this option be kept?

Reply:
Resolving for now, I'll put up another small PR to remove these options, and we can accept or deny it there instead of holding this up.